packages/nx/src/command-line/init/implementation/react/index.ts

```diff
@@ -32,41 +21,50 @@ type NormalizedOptions = Options & {
   pmc: PackageManagerCommands;
   appIsJs: boolean;
   reactAppName: string;
-  isCRA5: boolean;
-  npxYesFlagNeeded: boolean;
-  isVite: boolean;
   isStandalone: boolean;
 };
 
-export async function addNxToCraRepo(options: Options) {
-  if (!options.force) {
-    checkForUncommittedChanges();
+export async function addNxToCraRepo(_options: Options) {
+  if (!_options.force) {
     checkForCustomWebpackSetup();
   }
 
-  output.log({ title: '🐳 Nx initialization' });
+  const options = await normalizeOptions(_options);
+  await addBundler(options);
+
+  execSync(`echo "node_modules" >> .gitignore`, {
```

Comment:
We should be using platform agnostic `fs` utilities, not shell commands, for these file updates